librocketchat.c

```diff
@@ -3636,6 +3802,27 @@ rc_add_account_options(GList *account_options)
 	
 	option = purple_account_option_string_new(N_("Server Path"), "server_path", "");
 	account_options = g_list_append(account_options, option);
+
+#define ADD_VALUE(list, desc, v) { \
+	PurpleKeyValuePair *kvp = g_new0(PurpleKeyValuePair, 1); \
+	kvp->key = g_strdup((desc)); \
+	kvp->value = g_strdup((v)); \
+	list = g_list_prepend(list, kvp); \
+}
+
+
+	ADD_VALUE(encryption_values, _("No encryption"), "no_tls");
+	ADD_VALUE(encryption_values, _("Require encryption"), "tls");
+
+#undef ADD_VALUE

```

Comment:
Out of interest, was there a reason you went with a string in a dropdown vs a boolean? :)

Reply:
Mostly to keep it in line with other such menus, on the bitlbee side a drop down is also easier to see.